LayoutTests/fast/dom/normalize-doesnt-check-string-length.html

```diff
@@ -0,0 +1,25 @@
+<html>
+<head>
+    <script src="../../resources/js-test.js"></script>
+    <script>
+        function f1() {
+            const MAX_LENGTH = 1 << 28;
+
+            const largeString = 'A'.repeat(MAX_LENGTH);
+
+            const container = document.getElementById('container');
+
+            for (let i = 0; i < 9; ++i)
+                container.appendChild(document.createTextNode(largeString));
+
+            shouldThrowErrorName('container.normalize()', 'InvalidModificationError');
+            description("This test passes if webkit doesn't crash and there's an exception");
```

Comment:
s/webkit doesn't crash/there is no crash/ - both because of incorrect WebKit capitalization, and because the test is applicable to any browser engine.